provider/auth.md

```diff
@@ -0,0 +1,11 @@
+# Authorization
+
+Authorization for **provider** API should be provided via a JSON Web Token or [JWT](https://jwt.io/introduction/).
```

Comment:
Perhaps pivot this document away from discussing JWTs and towards specifying that authorization for the API will be communicated using Bearer Tokens, specified in the `Authorization` header, as detailed in [IETF RFC 6750](https://tools.ietf.org/html/rfc6750). Then a section can discuss how the spec does not dictate how those tokens are generated and transmitted to consumers, suggesting JWT as a common approach?